clients/ui/frontend/src/app/pages/modelCatalog/components/__tests__/ModelCatalogCard.test.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
```

Comment:
Are we planning to add cypress tests for these components or decided to keep the unit test for these component. Asking because the midstream has cypress testing and not unit test for this component.

Reply:
+1, if we use it as a cypress test is weird that you suddenly use unit testing